outport/process/transactionsfee/transactionsFeeProcessor.go

```diff
@@ -249,6 +271,19 @@ func (tep *transactionsFeeProcessor) prepareTxWithResultsBasedOnLogs(
 
 	for _, event := range txWithResults.log.GetLogEvents() {
 		if core.WriteLogIdentifier == string(event.GetIdentifier()) && !hasRefund {
+			if !check.IfNil(userTx) && tep.enableEpochsHandler.IsFlagEnabledInEpoch(common.FixRelayedBaseCostFlag, epoch) {
```

Comment:
same here the two code blocks inside prepareTxWithResultsBasedOnLogs and in prepareScrsNoTx are similar

Reply:
done in https://github.com/multiversx/mx-chain-go/pull/6546